salt/modules/zypperpkg.py

```diff
@@ -1428,7 +1410,6 @@ def install(
     skip_verify=False,
     version=None,
     ignore_repo_failure=False,
-    no_recommends=False,
```

Comment:
You cannot remove this as someone could possibly be calling these kwargs as positional args.

Also why not add `recommends=False` to the function, similar to `no_recommends`?

Reply:
Yes that's why I use `kwargs.get("no_recommends") is True `.

> Also why not add `recommends=False` to the function, similar to `no_recommends`?
I use kwargs but effectively I can add *recommends* to the function if your prefer (also same thing for *vendorchange*)

Reply:
> Yes that's why I use kwargs.get("no_recommends") is True 

This still would not cover the case where someone was calling the kwargs as positional args. Its not something they should be doing, but if we do not need to remove it I would rather keep the order to ensure that use case also works on upgrade.

> I use kwargs but effectively I can add recommends to the function if your prefer (also same thing for vendorchange)

I prefer to not use kwargs unless there is a valid use case when we need to use it.